cassava_trait.obo

```diff
@@ -8770,6 +8769,876 @@ relationship: variable_of CO_334:0013012 ! attieke-like aroma method
 relationship: variable_of CO_334:0103029 ! Attieke aroma scale 0-10
 created_by: Ebah Catherine
 
+[Term]
+id: CO_334:0003144
+name: Canopy formation Tricot block 3
```

Comment:
should 'Tricot block 3' be part of this variable? can it be made generic so others can reuse it?

Reply:
I guess you can still reuse the trait to create a variable that's not climmob specific.

Reply:
is there a way then to remove the Tricot language and 'block 3' ? I think it should be possible to define how traits like 'canopy formation' are assayed without specifying ''block 3". If someone else uses an array of 4 or 5 they should still use the same variable.

Reply:
In fact, we agreed while ago with Jacob  to not use the term 'TRICOT' but use  'comparative ranking method' instead with a  block number that indicates the number of varieties compared. So, " block:3" is an important information as it the number of technologies compared is part of the method used so it is part of the variable.